subworkflows/local/longread_consensus.nf

```diff
@@ -0,0 +1,35 @@
+//
+// Consensus calling for long reads
+//
+
+include { SAMTOOLS_CONSENSUS } from '../../modules/nf-core/samtools/consensus/main'
+include { MEDAKA } from '../../modules/nf-core/medaka/main'
+include { PIGZ_COMPRESS } from '../../modules/nf-core/pigz/compress/main'
+
+
+workflow LONGREAD_CONSENSUS {
+    take:
+    bam         // channel: [ val(meta), path(bam) ]
+    reference   // channel: [ val(meta), path(reference) ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    if ( params.longread_consensus_tool == 'medaka' ) {
+        input_medaka  = bam.combine(reference).map{ meta_bam, bam, meta_ref, ref -> [ meta_bam, bam, ref ]}
+        MEDAKA ( input_medaka )
+        ch_consensus = MEDAKA.out.assembly
+        ch_versions = ch_versions.mix(MEDAKA.out.versions)
+
+    } else if ( params.longread_consensus_tool == 'samtools' ) {
```

Comment:
Here samtools is being used for the consensus of nanopore reads but this is not aligned with what is stated in `nextflow_schema.json`

Reply:
Like i commented above, the test data is too small to run `medaka`.